lib/src/generator_frontend.dart

```diff
@@ -0,0 +1,377 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
```

Comment:
This file was `html_generator_instance.dart `. Most of the structure is maintained, but there's enough difference to convince Git that it's a new file.